samza-test/src/test/java/org/apache/samza/storage/kv/BlobStoreStateBackendIntegrationTest.java

```diff
@@ -82,8 +82,16 @@ public static Collection<Boolean> data() {
 
   private static final String IN_MEMORY_STORE_CHANGELOG_TOPIC = "inMemoryStoreChangelog";
 
-  private static final String LOGGED_STORE_BASE_DIR = new File(System.getProperty("java.io.tmpdir"), "logged-store").getAbsolutePath();
-  private static final String BLOB_STORE_BASE_DIR = new File(System.getProperty("java.io.tmpdir"), "blob-store").getAbsolutePath();
+  private static final String LOGGED_STORE_BASE_DIR;
+  private static final String BLOB_STORE_BASE_DIR;
+  static {
+    try {
+      LOGGED_STORE_BASE_DIR = Files.createTempDirectory("logged-store-").toString();
```

Comment:
This is unrelated cleanup. With this the temp base dir is unique for each test run (createTempDirectory adds a random suffix to the provided prefix).